projects/aca-content/folder-rules/src/rule-details/rule-details.ui-component.html

```diff
@@ -4,11 +4,11 @@
     <div class="aca-rule-details__form__row">
       <label class="aca-rule-details__name__label" for="rule-details-name-input">{{ 'ACA_FOLDER_RULES.RULE_DETAILS.LABEL.NAME' | translate }}</label>
       <div>
-        <mat-form-field appearance="fill" floatLabel='auto' class="aca-rule-details__form__row__field">
+        <mat-form-field appearance="fill" floatLabel='always' class="aca-rule-details__form__row__field">
+          <mat-label *ngIf="name.value === ''">{{ 'ACA_FOLDER_RULES.RULE_DETAILS.PLACEHOLDER.NAME' | translate }}</mat-label>
```

Comment:
can be `*ngIf="!name.value"` to cover null/undefined as well

Reply:
Added, thanks !